library/cmac.c

```diff
@@ -178,6 +178,7 @@ int mbedtls_cipher_cmac_starts(mbedtls_cipher_context_t *ctx,
 
     switch (type) {
         case MBEDTLS_CIPHER_AES_128_ECB:
+		case MBEDTLS_CIPHER_ARIA_128_ECB:
```

Comment:
Without looking any further at this, this indentation appears totally wrong

Reply:
Also, we should add `#if defined(MBEDTLS_ARIA_C)` (and similar for AES) around the switch cases to save on code size.

Reply:
@daverodgman added the same, just ignore my multiple empty commits, there were some issues on my side.

Reply:
Please add defines for AES and DES while you are there - thanks!